addOns/exportreport/src/main/java/org/zaproxy/zap/extension/exportreport/export/ReportExportPDF.java

```diff
@@ -0,0 +1,1111 @@
+        String labelURLs =
+                extensionExport.getMessages().getString("exportreport.export.message.pdf.urls");
+        String labelParameter =
+                extensionExport
+                        .getMessages()
+                        .getString("exportreport.export.message.pdf.parameters");
+        String labelAttack =
+                extensionExport.getMessages().getString("exportreport.export.message.pdf.attack");
+        String labelEvidence =
+                extensionExport.getMessages().getString("exportreport.export.message.pdf.evidence");
+        String labelOtherInfo =
+                extensionExport
+                        .getMessages()
+                        .getString("exportreport.export.message.pdf.otherinfo");
+        String labelSolution =
+                extensionExport.getMessages().getString("exportreport.export.message.pdf.solution");
+        String labelReferences =
+                extensionExport
+                        .getMessages()
+                        .getString("exportreport.export.message.pdf.references");
+        String labelCWEID =
+                extensionExport.getMessages().getString("exportreport.details.cweid.label");
+        String labelWASCID =
+                extensionExport.getMessages().getString("exportreport.details.wascid.label");
+        String labelReqHeader =
+                extensionExport.getMessages().getString("exportreport.details.requestheader.label");
+        String labelRespHeader =
+                extensionExport
+                        .getMessages()
+                        .getString("exportreport.details.responseheader.label");
+        String labelReqBody =
+                extensionExport.getMessages().getString("exportreport.details.requestbody.label");
+        String labelRespBody =
+                extensionExport.getMessages().getString("exportreport.details.responsebody.label");
+
+        // detailsToInclude is used to filter out details not wanted by the user
+        ArrayList<String> detailsToInclude = extensionExport.getIncludedAlertDetails();
+
+        Alert alert = alerts.get(0);
+
+        page = new PDPage(pageSize);
+        document.addPage(page);
+
+        // calculate initial positioning on the page (origin = bottom left)
+        // Point2D.Float textInsertionPoint = new Point2D.Float(page.findMediaBox().getLowerLeftX()
+        // + marginPoints, page.findMediaBox().getUpperRightY() - marginPoints);
+        Point2D.Float textInsertionPoint = getPageInitialInsertionPoint();
+
+        textInsertionPoint =
+                addText(alertCategoryLabelFormatting, alert.getName(), textInsertionPoint);
+
+        // Not added if not selected as detail to include
+        if (detailsToInclude.contains(
+                extensionExport
+                        .getMessages()
+                        .getString("exportreport.export.message.pdf.description"))) {
```

Comment:
These should use the keys `exportreport.details` instead (while most of them are equal some aren't, e.g. `Other Info` vs `Other Information`).